app/docs/md/learn/concepts/styling/customization.md

```diff
@@ -1,28 +1,45 @@
 ---
-title: Customization
+title: Customizing Styles
 ---
 
-It is possible to completely customize your project's utility classes. To do so you will need to do two things:
+It is possible to completely customize your project's utility classes. There are two ways to do this:
 
-## 1. JSON config file
+## enhance.json
 
-Add a `styleguide.json` (or choose another name) file to the root of your project.
+The `enhance.json` file at the root of your project is designed to act as [a single source of configuration for Enhance plugins](/docs/learn/starter-project/configuration).
 
-<doc-link-callout link="https://raw.githubusercontent.com/enhance-dev/enhance-styles/main/config.json" mark="📄">
-  Copy the default config to get started
+To add configuration options for Enhance Styles, simply add the key `"@enhance/styles"` under the `plugins` object. For example:
+
+<doc-code filename="enhance.json">
+
+```json
+{
+  "plugins": {
+    "@enhance/styles": {
+      "fonts": {
+        "sans": "system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif",
+        "serif": "Georgia,Cambria,Times New Roman,Times,serif",
+        "mono": "Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace"
+      },
+      …
+    }
+  }
+}
+```
+
+</doc-code>
+
+<doc-link-callout link="https://github.com/enhance-dev/enhance-styles#readme" mark="🎛️">
+  Check out the Enhance Styles readme for a full list of the available customizations
```

Comment:
We'll need to update the Enhance Styles readme slightly (along with the repo itself) before shipping this docs update.